CHANGELOG.md

```diff
@@ -0,0 +1,32 @@
+## moler 1.3.0
+
+* Add entering to UNIX_LOCAL state when device is closing
+* Add devices when load config
+* Fix first moler prompt catch
+* Fix iperf mbits calculation
+* Fix iperf command
+* Add UNIX devmem cmd
+* Remove properly new line chars on some connections
+
+
+## moler v1.1.0
+
+* Add gzip command
+* skip asyncio test under python 2
+* Textual events chunk processing
+* ret is not required for sudo
```

Comment:
### Added
* new UNIX command handled by moler: **gzip**
* skip asyncio test under python 2 (internal CI of moler)

### Fixed
* wrong processing of chunked lines by textual events
* sudo command incorrectly handling result of sudo-embedded command